pkgs/development/mobile/android-image-utils/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "android-image-utils";
+  version = "2012-08-08";
+
+  src = fetchFromGitHub {
+    owner = "freesmartphone";
+    repo = "utilities";
+    rev = "1097722a3e5c92e6bef32c1c5eb75f199411c944";
+    sha256 = "16qwiazik6yqnp2bdrmg603ddz62xk75xqjn5272c4mw5f7kv2q1";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ zlib ];
+
+  postPatch = "cd android/image-utils && ls";
```

```suggestion
  postPatch = "cd android/image-utils";
```

Also I think this is not the correct position to change directories but I am not sure how to do it correctly.

---

I removed the ``ls``.
The position to change dirs was copied from some other packages.
If another position is better, let me know.

---

You can use `sourceRoot`.

---

Thanks, done.